en/complete_vehicles/atl_mantis_edu.md

```diff
@@ -0,0 +1,211 @@
+2. Power up the drone.
+   It will reboot and update automatically.
+3. Wait until the WiFi re-appears and re-connect.
+
+Note: if the file has been removed from the SD card, the camera has successfully been updated.
+
+### Update Gimbal
+
+1. Place the gimbal image **gimbal.yuneec** on the SD card.
+2. Power up the drone.
+   The gimbal will be automatically updated.
+3. After the update, you might hear a faint beep as the gimbal starts up again.
+
+:::note
+The file will be renamed to **gimbal.yuneec.updated** if the update was successful.
+If not, you should find a file **gimbal_update.log** with more details.
+:::
+
+### Update Autopilot (PX4)
+
+1. Place the **autopilot.px4** file on the SD card.
+2. Power up the drone.
+   The autopilot will be automatically updated.
+3. After the update, you should hear PX4 starting up by the usual beeps.
+
+:::note
+The file will be renamed to **autopilot.px4.updated** if the update was successful.
+If not, you should find a file **autopilot_update.log** with more details.
+:::
+
+### Update RC
+
+1. Place the **update.lzo** file on a USB stick (or SD card in a USB card reader). It needs to be FAT32 formatted.
+2. Plug the USB stick into the USB-A (not USB-C) port of the RC controller.
+3. Power on the RC controller by long pressing the power button.
+4. Wait for the blue LED to be on or blinking. This means the controller has finished booting.
+5. Hold the Home/RTL button (on the left) pressed while pressing the photo button (top right) quickly 4 times (within 2 seconds).
+6. Release the buttons when you hear a beep. It will beep while applying the update.
+7. Once the beeping stops with a positive tone (after about 90 seconds), the controller reboots automatically.
+
+:::note
+The **update.lzo** file is not remove on the USB stick after the update.
+:::
+
+## Developing PX4 Firmware
+
+### Build the PX4 Firmware
+
+To build PX4 for the Mantis, use the make command:
+```
+make atl_mantis-edu
+```
+
+### Upload/flash the PX4 Firmware
+
+There are two ways to update the firmware:
+
+#### Upload while connected to the Mantis' WiFi:
+
+The firmware will be uploaded to the SD-card over http using curl:
+```
+make atl_mantis-edu upload_wifi
+```
+
+:::note
+Make sure to have an SD card inserted for this.
+:::
+
+#### Upload using the SD card
+
+The built firmware can be found in `build/atl_mantis-edu_default/atl_mantis-edu_default.px4`.
+
+To update the vehicle with this firmware: rename the file to **autopilot.px4**, copy it to root of the SD card, and restart the vehicle (as described above).
+
+
+## Using MAVSDK with the Mantis
+
+By default MAVSDK connects via MAVLink on UDP port 14540.
+This port is configured for the Mantis, so MAVSDK examples and tests should mostly work out of the box.
+
+TODO: Add more detailed tutorial how to use MAVSDK with the Mantis.
+
+## Connection
+
+You can either connect to the drone using WiFi, or using a USB cable connected to the remote control.
```

Comment:
We have got to here before it is clear that we can connect to QGC. I think it is probably worth stating here or in the intro that this is possible. I also think it is worth being clear what you can do with QGC after update - can I use it for firmware update (after the first time)? Can I set parameters, stream video?  Can I update the camera?

Here you might start with something like.

> You can connect to the drone for firmware update or for connecting to QGC or MAVSDK (or other mavlink client) ...

Reply:
> can I use it for firmware update (after the first time)?

No you cannot.